frontend/src/scenes/notebooks/Nodes/NodeWrapper.tsx

```diff
@@ -336,7 +346,7 @@ export function createPostHogWidgetNode<T extends CustomNotebookNodeAttributes>(
         name: wrapperProps.nodeType,
         group: 'block',
         atom: true,
-        draggable: true,
+        draggable: false,
```

Comment:
If you make this "draggable: false, selectable: false" then you basically get the control we were looking for. I have an alternative branch I was playing with where i had some realisations about this overall approach that we should probably chat about 👍